src/shared/locales/ja/preferences.js

```diff
@@ -0,0 +1,37 @@
+export default {
+  'basic': '基本設定',
+  'advanced': '詳細設定',
+  'lab': '実験室',
+  'save': '保存して適用',
+  'discard': '放棄',
+  'startup': '起動',
+  'auto-resume-all': '起動後自動的に未完了タスクを再開',
+  'default-dir': 'デフォルトディレクトリ',
+  'mas-default-dir-tip': 'App Store の Sandbox 制限のため，デフォルトディレクトリ設定は「Download」を推奨します。',
+  'task-manage': 'タスク管理',
+  'max-concurrent-downloads': '最大同時タスク数',
+  'max-connection-per-server': '最大サーバ接続数',
+  'new-task-show-downloading': '新規タスクを作成後自動的にタスク画面に移る',
+  'continue': '続ける',
+  'task-completed-notify': 'タスク完了後に通知する',
+  'auto-purge-record': 'アプリケーション終了後自動的にタスク履歴を削除',
+  'ui': 'UI',
```

Comment:
Something added here, please sync the latest master code.
https://github.com/agalwood/Motrix/blob/ee78ae262e97fbe980399847861ab0948bcb87a1/src/shared/locales/en-US/preferences.js#L19-L22